app/views/notifications/_badgeachievement.html.erb

```diff
@@ -12,7 +12,7 @@
       <p class="color-base-70"><em><%= json_data["badge_achievement"]["rewarding_context_message"].html_safe %></em></p>
     </div>
 
-  <p><a href="<%= json_data["user"]["path"] %>" class="crayons-btn w-100 m:w-50">Visit your profile</a></p>
+  <p class="self-stretch"><a href="<%= json_data["user"]["path"] %>" class="crayons-btn w-100 m:w-50">Visit your profile</a></p>
```

Comment:
That's one bug I've found accidentally and that is completely unrelated to this PR but I sneaked it in :P